cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsStackRequestHelper.java

```diff
@@ -98,7 +100,8 @@ public DeleteStackRequest createDeleteStackRequest(String cFStackName) {
         if (awsInstanceProfileView.isInstanceProfileAvailable()) {
             parameters.add(new Parameter().withParameterKey("InstanceProfile").withParameterValue(awsInstanceProfileView.getInstanceProfile()));
         }
-        if (ac.getCloudContext().getLocation().getAvailabilityZone().value() != null) {
+        if (ac.getCloudContext().getLocation().getAvailabilityZone() != null
```

Comment:
is this possible that the ac.getCloudContext().getLocation().getAvailabilityZone()  null ?

Reply:
Yes, for the same reason as above.